pkg/cfn/template/matchers/matchers.go

```diff
@@ -0,0 +1,333 @@
+package template
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/onsi/gomega"
+	"github.com/onsi/gomega/types"
+
+	. "github.com/weaveworks/eksctl/pkg/cfn/template"
+)
+
+func matchTemplate(actualTemplate interface{}) error {
```

Comment:
Since this is not a matcher how about calling it "checkIsTemplate()"?

Reply:
Good idea, thanks!